doc/source/user_guide/missing_data.rst

```diff
@@ -91,6 +91,10 @@ See :ref:`integer_na` for more.
 
 Datetimes
 ---------
+.. note::
+   If you are adding type checking to your application, you may need access to ``NaTType`` and ``NAType``.
+
+   ``NaTTYPE`` and ``NAType`` have now been added and can be imported from "pandas.api.typing".
```

Comment:
you don't need this sentence here

Reply:
> you don't need this sentence here

Which sentence are you referring to?

Reply:
> you don't need this sentence here
If you mean the entire comment would you suggest somewhere else where it fits better?